docs/guides/server/logging.adoc

```diff
@@ -300,7 +300,14 @@ You can use regular expressions to exclude them, such as:
 
 In this case, all calls to the `/realms/my-internal-realm/` and subsequent paths will be excluded from the HTTP Access log.
 
-<@opts.printRelevantOptions includedOptions="log log-*" excludedOptions="log-console-* log-file log-file-* log-syslog-* log-mdc*">
+[[logs-export-opentelemetry]]
+== Logs export (OpenTelemetry)
+
+It is possible to export OpenTelemetry logs from your deployment to the OpenTelemetry collector for centralized log management.
+
+For more details, see the _Centralize your observability stack with OpenTelemetry_ guide.
+
+<@opts.printRelevantOptions includedOptions="log log-*" excludedOptions="log-console-* log-file log-file-* log-syslog-* log-mdc* log-export-*">
```

Comment:
This feels fragile. Should we rather use `includedOptions="log-export-*"`?

Reply:
Done